7.application_configuration.md

```diff
@@ -70,70 +68,61 @@ The specification defines three major things: Parameters that can be overridden
 
 | Attribute | Type | Required | Default Value | Description |
 |-----------|------|----------|---------------|-------------|
-| `variables` | [`[]Variable`](#variable) | N | | Variables that can be referenced in parameter values and properties. |
-| `scopes` | [`[]Scope`](#scope) | N | | Application scope definitions. If defined, scopes are accessible by components from any other application configurations. |
```

Comment:
Is it necessary to remove scope?  It seems that it will be too verbose in not too complicated application like below without it.
```yaml
componentA
    scope :
         - Kind : K1
            Name : n1
        -  Kind : K1
            Name : n2
componentB
    scope :
         - Kind : K1
            Name : n1
        -  Kind : K2
            Name : n2
componentC
    scope :
        - Kind : K1
            Name : n2
        -  Kind : K2
            Name : n2

```

Reply:
I'll put it back as you said

Reply:
In alpha1, the scopes object here (as a sibling of components) defined a scope _instance_. I don't understand why we'd have a list of scope _references_ here. Where are the scope instances created and why would I reference a list of them at the root of the application configuration?

Reply:
I also assumed we'd remove this `scopes` field for the reason @vturecek mentioned; because it was previously used to _instantiate_ scopes, not associate them with an application. I can see the rationale in repurposing the field as an array of
scopes in which all of an application's workloads will exist, though. If we do this it should be an array of references to scopes, not embedded instances of scopes.

Reply:
Maybe there's some misunderstandings here. The scopes section is used for index ScopeRefs, so we don't need to repeatedly write scopeRef under component section. It's just for convenient.

Reply:
Just clarifying here.

This PR removes the **instantiation** of scope instances in AppConfig.

@ryanzhang-oss is suggesting the **scope refs** are too verbose.

@vturecek is asking why there are **scope refs** here? Answer: because a component needs to belong a scope, and it does so by adding a scope ref.

@negz is assuming we are removing this scopes field. And yes we do.